libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationRouteProcessor.java

```diff
@@ -38,7 +34,8 @@
   private CurrentLegAnnotation currentLegAnnotation;
   private Geometry routeGeometryWithBuffer;
 
-  RouteProgress buildNewRouteProgress(MapboxNavigator navigator, NavigationStatus status, DirectionsRoute route) {
+  RouteProgress buildNewRouteProgress(MapboxNavigator navigator, NavigationStatus status,
```

Comment:
Could we place each param on the new line?)

Reply:
This is Java we can wait until `NavigationRouteProcessor` conversion.

Reply:
Right, sorry, my fault 😞